build.py

```diff
@@ -38,3 +42,71 @@ def initialize(project):
 
     project.get_property('filter_resources_glob').extend(
         ['**/c_bastion/__init__.py'])
+
+
+@init(environments='travis')
+def set_properties_for_travis_builds(project):
+    project.version = '%{0}.%{1}'.format(
+        project.version, os.environ.get('TRAVIS_BUILD_NUMBER', 0))
```

Comment:
Personally I would put this into the original definition of `version` so that we always get an `X.Y` style version with `Y` either 0 for personal builds or a longer number for automated builds.